torchvision/transforms/transforms.py

```diff
@@ -623,7 +640,7 @@ def __call__(self, img):
         Returns:
             PIL Image: Random perspectivley transformed image.
         """
-        if not F._is_pil_image(img):
+        if not F.F_pil._is_pil_image(img):
```

Comment:
This looks weird? Although I see the point, I'm not sure I would recommend users importing `F_pil` from the `functional.py` file.

Plus, both the `functional_pil` and `functional_tensor` are meant to be just implementation details functions, and might be preferable if they didn't leak to `transforms.py`

Reply:
I agree, looks weird. I can introduce in `functional.py` as shortcut 
```python
from . import functional_pil as F_pil
from . import functional_tensor as F_t


_is_pil_image = F_pil._is_pil_image
```

Reply:
sound good to me